modules/statics/src/coins.ts

```diff
@@ -4725,6 +4741,14 @@ export const coins = CoinMap.fromCoins([
     '0xaea8e1b6cb5c05d1dac618551c76bcd578ea3524',
     UnderlyingAsset.SGR
   ),
+  erc20(
+    '518fe28c-87fc-4b9f-b07e-a2b8f63f4686',
+    'sgt',
+    'Suzu Governance Token',
```

Comment:
In contract, the name of the token seems to be `SGT`, Double checking if this is correct?

Reply:
Yes, it's added as sgt only.